crates/biome_cli/src/commands/mod.rs

```diff
@@ -201,11 +201,21 @@ pub enum BiomeCommand {
         ///
         /// - When a rule group is passed, the `recommended` flag is enabled, but if the `all` flag is enabled.
         ///
-        /// Example: `biome lint --rule=correctness/noUnusedVariables`
+        /// Example: `biome lint --only=correctness/noUnusedVariables --only=suspicious`
+        #[bpaf(long("only"), argument("GROUP|RULE"))]
+        only: Vec<RuleSelector>,
+
+        /// Skip the given rule or rule group.
+        ///
+        /// The option overrides the Biome configuration file as follows:
+        ///
+        /// - When a rule is passed, its severity level is set to `off'.
+        ///
+        /// - When a rule group is passed, all rules of the group are skipped.
```

Comment:
> Also, I think we should align the documentation of this flag with its counter part. The closer they are, the easier to explain things and less maintenance for us.

Unfortunately, `--only` is fundamentally different from `--skip` because it doesn't force all the rules in a group to be executed. Do you think we should change it?

Reply:
I would expect both to behave in the same way, logically speaking.